apps/server/src/shared/domain/entity/task-card.entity.ts

```diff
@@ -39,10 +42,20 @@ export class TaskCard extends BaseEntityWithTimestamps implements ICard, ITaskCa
 	@Property()
 	draggable = true;
 
+	@Property()
+	visibleAtDate: Date;
+
+	@Property()
+	dueDate: Date;
+
 	public getCardElements() {
 		return this.cardElements.getItems();
 	}
 
+	public isVisibleBeforeDueDate() {
```

Comment:
this could develop into "isVisible", adding logic as hidden prop, draft, visible for user (e.g. creator can see, others not) etc